tests/rules/flowbits/test.rules

```diff
@@ -0,0 +1,5 @@
+alert ip any any -> any any (msg:"isset rule need an option"; flowbits:isset; sid:1;)
+alert ip any any -> any any (msg:"isnotset rule need an option"; flowbits:isnotset; sid:2)
```

Comment:
In addition to the lack of a name for the flowbits variable, this rule would also result in a parse error because it lacks the `;` after the rule sid. Each field within the parenthesis must be followed by those. ;)